signer/local/local.go

```diff
@@ -336,14 +326,23 @@ func (s *Signer) Sign(req signer.SignRequest) (cert []byte, err error) {
 		}
 	}
 
+	var distPoints = safeTemplate.CRLDistributionPoints
+	if distPoints != nil && len(distPoints) > 0 {
+		safeTemplate.CRLDistributionPoints = distPoints
```

Comment:
It might just be not having had enough coffee yet, but I don't see what this is doing.

Reply:
I blindly copied this from `Signer.sign` but yeah you're right, this doesn't appear to actually _do anything_. It appears [when this was introduced](https://github.com/cloudflare/cfssl/commit/6728e3be9365e6fa8bc699014a2046ca6c82c27a#diff-8d79bfc4bb0987be2c41ab248b24b30fR101) the `if` was after `FillTemplate`, which allowed the profile `CRLDistributionPoints` to be overridden by those in the template. In #767 it appears to have been moved before `FillTemplate` where it now does nothing.

Reply:
(If the overriding behavior is something we wish to preserve I'm happy to move it back to after `FillTemplate` in this PR.)